lib/sidekiq/launcher.rb

```diff
@@ -171,7 +174,7 @@ def ❤
             conn.hmset(key, "info", to_json,
               "busy", curstate.size,
               "beat", Time.now.to_f,
-              "rtt_us", rtt,
+              "rtt_us", 0,
```

Comment:
I wonder if it would be worth not storing this in redis at all?
it looks like this is only used in `api.rb` and our fork could stop retrieving it and converting it to an int

https://github.com/braze-inc/sidekiq/blob/0f7f5047cf852ea3c9f0bdcb8b46bac9b22215ec/lib/sidekiq/api.rb#L794-L810

Reply:
There's a test that relies on it so I figured at some point he may add something else that does, and I thought it'd be the easiest change to use 0. I don't think it matters from a RAM perspective.